psydac/linalg/tests/test_iterative_solvers.py

```diff
@@ -0,0 +1,498 @@
+import numpy as np
+import pytest
```

Comment:
Please add an empty line before the Psydac imports. See [PEP 8](https://peps.python.org/pep-0008/#imports) as a reference.

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)